RZBluetooth/Command/RZBCommandDispatch.h

```diff
@@ -89,6 +89,11 @@
  */
 - (void)dispatchCommand:(RZBCommand *)command;
 
+/**
+ * Submit a new command with custom timeout.
+ */
+- (void)dispatchCommand:(RZBCommand *)command timeout:(NSTimeInterval) timeout;
```

Comment:
Spacing is inconsistent with the rest of the parameters in this file.

```suggestion
- (void)dispatchCommand:(RZBCommand *)command timeout:(NSTimeInterval)timeout;
```